package.json

```diff
@@ -46,21 +46,21 @@
     "typescript": "^4.6.2"
   },
   "scripts": {
-    "prepare": "npm run build",
-    "bundle": "rollup -c",
     "build": "npm run bundle && npm run tsc",
     "build:debug": "npm run bundle -- -m && npm run tsc",
+    "build:keys": "node tools/build-keys-from-ts",
+    "bundle": "rollup -c",
+    "coverage": "c8 report --reporter lcov && opener coverage/lcov-report/index.html",
```

Comment:
Not sure about this one, maybe `test:open-coverage`?

Reply:
Makes sense.